<launch>

  	<node 	pkg="obstacle_estimator"
    		type="obstacle_filter_node"
    		name="obstacle_filter_node_1"
    		respawn="false"
    		launch-prefix=""
    		required="true">
			<remap from="/obstacle/pose" to="/Target1/pose"/>
			<remap from="/obstacle/state_estimation" to="/Target1/state_estimation"/>
  	</node>

	<node 	pkg="obstacle_estimator"
			 type="obstacle_filter_node"
			 name="obstacle_filter_node_2"
			 respawn="false"
			 launch-prefix=""
			 required="true">
			 <remap from="/obstacle/pose" to="/Target2/pose"/>
			 <remap from="/obstacle/state_estimation" to="/Target2/state_estimation"/>
	</node>

</launch>
